build/Build.cs

```diff
@@ -36,12 +36,14 @@ class Build : NukeBuild
 
     AbsolutePath SourceDirectory => RootDirectory / "source";
     AbsolutePath ArtifactsDirectory => RootDirectory / "artifacts";
+    AbsolutePath TestResultsDirectory => RootDirectory / "TestResults";
```

Comment:
Standardising `TestResults` to have consistent (Octopus repos) path for GHA `unit test report` workflow